mailpoet/assets/js/src/automation/integrations/core/steps/delay/edit.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  PanelBody,
+  TextControl,
+  SelectControl,
+  Flex,
+  FlexItem,
+} from '@wordpress/components';
+import { dispatch, useSelect } from '@wordpress/data';
+import { PlainBodyTitle } from '../../../../editor/components/panel';
+import { store } from '../../../../editor/store';
+import { DelayTypeOptions } from './types/delayTypes';
+
+export function Edit(): JSX.Element {
+  const { selectedStep } = useSelect(
+    (select) => ({
+      selectedStep: select(store).getSelectedStep(),
+    }),
+    [],
+  );
+
+  return (
+    <PanelBody opened>
+      <PlainBodyTitle title="Wait for" />
+      <Flex align="top">
+        <FlexItem style={{ flex: '1 1 0' }}>
```

Comment:
Inline styles... I guess in this case, it's acceptable. Just thinking out loud. I'm surprisedWP components have such primitives, TIL, and grid seems to be experimental, as far as I can see.

Using a single parent `<div>` for the `<TextControl>` elements with a bit of CSS (display `grid` and `gap`, if needed) would simplify it a bit. But not important...